ietf-layer0-types-ext.yang

```diff
@@ -622,20 +650,24 @@ module ietf-layer0-types-ext {
          sample the function penalty = f(CD, PMD).";
       uses cd-pmd-penalty ;
     }
+
     leaf max-diff-group-delay  {
            type int32;
            config false;
            description "Maximum Differential group delay of this mode
                         for this lane";
     }
-    leaf max-polarization-dependent-loss {
-      type decimal64 {
-        fraction-digits 2;
-      }
-      units dB ;
+
+    list max-polarization-dependent-loss-penalty {
+      config false;
       description
-        "Maximum acceptable accumulate polarization dependent loss";
+        "Optional penalty associated with the maximum acceptable 
+        accumulated polarization dependent loss.
+         This list of pair pdl and penalty can be used to
+         sample the function penalty = f(pdl).";

```

Comment:
From the initial proposal, I though that the function was pdl = f(penalty):

https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/issues/41#issue-723272062

@ggalimba56 : could you please clarify?

Reply:
```suggestion
        "Optional penalty associated with the maximum acceptable 
        accumulated polarization dependent loss.
         This list of pair pdl and penalty can be used to
         sample the function pdl = f(penalty).";
```